### PR TITLE
Feedbin new sources

### PR DESCRIPTION
**Pull request by jcortes**
# Actions
- [x] Get Subscriptions | Return all subscriptions. [See the docs here](https://github.com/feedbin/feedbin-api/blob/master/content/subscriptions.md#get-subscriptions).

# Sources
- [x] New Search | Emit new event when a new search is created. [See the docs here](https://github.com/feedbin/feedbin-api/blob/master/content/saved-searches.md#get-saved-searches)
- [x] Liked Item | Emit new event when a new item is liked. [See the docs here](https://github.com/feedbin/feedbin-api/blob/master/content/starred-entries.md#get-starred-entries)
- [x] Saved Item | Emit new event when a new item is saved. [See the docs here](https://github.com/feedbin/feedbin-api/blob/master/content/entries.md#get-v2entriesjson)

<a href="https://gitpod.io/#https://github.com/PipedreamHQ/pipedream/pull/3216"></a>

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible
- [ ] [Create subscription webhooks sources (vs polling sources) whenever possible](https://pipedream.com/docs/components/guidelines/#webhook-vs-polling-sources)
- [ ] [Always emit a `summary` with events](https://pipedream.com/docs/components/guidelines/#emit-a-summary)
- [ ] [Use a built-in deduping strategy wherever possible](https://pipedream.com/docs/components/guidelines/#deduping)
- [ ] When building webhook sources, [follow these best practices](https://pipedream.com/docs/components/guidelines/#webhook-sources)
- [ ] When building polling sources, [follow these best practices](https://pipedream.com/docs/components/guidelines/#polling-sources)

File: components/feedbin/sources/saved-item/saved-item.mjs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      propDefinition: [
+        feedbin,
+        "feedId",
+      ],
+    },
+  },
+  async run() {
+    const { feedId } = this;
+
+    const lastCreatedAt = this.db.get("lastCreatedAt");
```
Comment (andrewjschuang) [inline components/feedbin/sources/saved-item/saved-item.mjs:30]: Can we create a `db` helper method here to maintain the guidelines?

File: components/feedbin/sources/saved-item/saved-item.mjs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        since: lastCreatedAt,
+      },
+    });
+
+    const [
+      lastEntry,
+    ] = feedEntries;
+
+    if (lastEntry) {
+      this.db.set("lastCreatedAt", lastEntry.created_at);
```
Comment (andrewjschuang) [inline components/feedbin/sources/saved-item/saved-item.mjs:46]: Can we create a `db` helper method here to maintain the guidelines?

**Review by andrewjschuang — DISMISSED** [review]
@jcortes thanks for the submission!

I've added a few comments.
I think we could create a `db` helper method and support pagination in the sources.

**Review by vellames-turing — APPROVED** [review]
lgtm

**Comment by vunguyenhung** [comment]
Hi everyone, all test cases are passed! Ready for release! 

Test report
[FeedBin_3216.pdf](https://github.com/PipedreamHQ/pipedream/files/9072492/FeedBin_3216.pdf)

**Review by vellames-turing — APPROVED** [review]

**Pull request merged**